openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -578,6 +578,28 @@ response_type=code
 &authorization_details=...
 ```
 
+If the request is successful, the  Authorization Server MUST respond with a 201 Created status code and the Location header field containing the URL of the Authorization Request URI.
```

Comment:
text mentions `Location` header but missing in the example

Reply:
This was changed in the meantime.